.github/workflows/aws-deploy.yaml

```diff
@@ -47,6 +56,12 @@ on:
       SLACK_WEBHOOK:
         required: true
 
+      HL_GITHUB_TOKEN:
+        required: false
+
+      SECRET_ENVIRONMENT:
```

Comment:
What does it represent ? Can you add a comment pls?

Reply:
Yes. HL_GITHUB_TOKEN should be defined only you have private submodules to make it able to sync them. Action check if it set and if no - just checkout repository, if yes - also syncs submodules. That's because we use private submodules in user-service

Reply:
I meant `SECRET_ENVIRONMENT`